<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_fbnwb955mi/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -14,6 +14,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py<39]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
@@ -25,8 +26,7 @@ requirements:
     - python
     - pip
     - setuptools >=77
-    - wheel
-    - zlib
+    - zlib {{ zlib }}
   run:
     - python
     - async-timeout

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -57,11 +57,8 @@ test:
     - mv gh_src/tests tests    # [not win]
     # Configure flags and docker image name (for unix only)
     # see: https://github.com/aio-libs/aiokafka/blob/master/Makefile#L3-L6
-    - export FLAGS=--maxfail=3    # [not win]
-    - export SCALA_VERSION=2.13   # [not win]
-    - export KAFKA_VERSION=2.8.1  # [not win]
-    - export DOCKER_IMAGE=aiolibs/kafka:$(SCALA_VERSION)_$(KAFKA_VERSION)  # [not win]
-    - export LOG_FORMAT="%(asctime)s %(levelname)s %(message)s"            # [not win]    
+    - set FLAGS=--maxfail=3     # [win]
+    - export FLAGS=--maxfail=3  # [not win]
     # Invoke tests like from the Makefile
     # see: https://github.com/aio-libs/aiokafka/blob/master/Makefile#L59
     #
@@ -69,7 +66,14 @@ test:
     # Docker is not available on Windows, do not pass a docker image
     # to skip its tests (they are fine on unix).
     - pytest -s -v --log-level DEBUG %FLAGS% tests    # [win]
-    - pytest -s -v --log-format="$LOG_FORMAT" --log-level DEBUG --docker-image=$(DOCKER_IMAGE) $(FLAGS) tests  # [not win]
+    # We decided to skip the docker tests for unix as well on our build system.
+    # To run tests locally comment this line and un-comment the lines below it.
+    - export LOG_FORMAT="%(asctime)s %(levelname)s %(message)s"                 # [not win]    
+    - pytest -s -v --log-format="$LOG_FORMAT" --log-level DEBUG $(FLAGS) tests  # [not win]
+    # - export SCALA_VERSION=2.13   # [not win]
+    # - export KAFKA_VERSION=2.8.1  # [not win]
+    # - export DOCKER_IMAGE=aiolibs/kafka:$(SCALA_VERSION)_$(KAFKA_VERSION)  # [not win]
+    # - pytest -s -v --log-format="$LOG_FORMAT" --log-level DEBUG --docker-image=$(DOCKER_IMAGE) $(FLAGS) tests  # [not win]
 
 about:
   home: https://github.com/aio-libs/aiokafka

